sycl/doc/extensions/experimental/sycl_ext_oneapi_raw_kernel_arg.asciidoc

```diff
@@ -108,15 +115,15 @@ namespace sycl::ext::oneapi {
 
   class raw_kernel_arg {
   public:
-    raw_kernel_arg(void* bytes, size_t count);
+    raw_kernel_arg(const void* bytes, size_t count);
   };
 
 } // namespace sycl::ext::oneapi
 ----
 
 [source,c++]
 ----
-raw_kernel_arg(void* bytes, size_t count);
+raw_kernel_arg(const void* bytes, size_t count);
```

Comment:
We had a long offline discussion about the ownership of the `bytes` memory and decided that `raw_kernel_arg` should be a "view" to that memory, meaning that it does not make a copy.  I see that the description already documents this as a "view".

I think it would be good to emphasize this a bit, though.  Let's add a sentence like this to the end of the _Effects_ paragraph:

> Since the `raw_kernel_arg` object is only a view, the caller must ensure that the lifetime of the `bytes` memory lasts at least as long as the lifetime of the `raw_kernel_arg` object.

Reply:
Good idea. I have added it to the effects of the ctor. I assumed that was the one you meant.

Reply:
Yes, Perfect.  Thanks!